infra/helm/meshforms/templates/NOTES.txt

```diff
@@ -0,0 +1,22 @@
+1. Get the application URL by running these commands:
```

Comment:
Is this used?

Reply:
I guess this is part of the template

Reply:
NOTES.txt is printed at the end of `helm install`? We don't really need it for internal charts, I suppose. 

https://helm.sh/docs/chart_template_guide/notes_files/